control-plane/pkg/reconciler/channel/controller.go

```diff
@@ -89,10 +90,12 @@ func NewController(ctx context.Context, watcher configmap.Watcher, configs *conf
 		)
 	}
 
+	caCerts, err := reconciler.getCaCerts()
 	impl := kafkachannelreconciler.NewImpl(ctx, reconciler)
 	IPsLister := prober.IdentityIPsLister()
 	reconciler.Prober = prober.NewAsync(ctx, http.DefaultClient, "", IPsLister, impl.EnqueueKey)
 	reconciler.IngressHost = network.GetServiceHostname(configs.IngressName, configs.SystemNamespace)
+	reconciler.NewProber, err = prober.NewComposite(ctx, configs.IngressPodPort, IPsLister, impl.EnqueueKey, &caCerts)
```

Comment:
```suggestion
	reconciler.Prober, err = prober.NewComposite(ctx, configs.IngressPodPort, configs.IngressPodTlsPort, IPsLister, impl.EnqueueKey, &caCerts)
```

Reply:
We also need to check the error here

Reply:
There configs.ingresspodtlsport giving undefeated error there ?